dask_ml/decomposition/pca.py

```diff
@@ -327,11 +327,11 @@ def _fit(self, X):
         # Compute noise covariance using Probabilistic PCA model
         # The sigma2 maximum likelihood (cf. eq. 12.46)
         self.noise_variance_ = 0.0
-        if n_components < min(self.n_features_, self.n_samples_):
+        if n_components < min(self.n_features_in_, self.n_samples_):
             if solver == "randomized":
                 self.noise_variance_ = (
                     total_variance - self.explained_variance_.sum()
-                ) / (min(self.n_features_, self.n_samples_) - n_components)
+                ) / (min(self.n_features_in_, self.n_samples_) - n_components)
```

Comment:
Should this renaming of `n_features_` to `n_features_in_` trigger a deprecation warning?

Reply:
scikit-learn is handling that. In this case, `sklearn.preprocessing.OneHotEncoder` will raise a warning.